doc/nrf/releases/release-notes-2.2.0.rst

```diff
@@ -0,0 +1,948 @@
+    * P-GPS partition section to the :file:`ncs/nrf/subsys/partition_manager/Kconfig` file.
+    * Board-specific static Partition Manager configuration shared among build types.
+      The configuration can be defined in the board's directory.
+
+  * Updated the :file:`ncs/nrf/subsys/partition_manager/pm.yml.pgps` file to place P-GPS partition in external flash when so configured.
+
+  * Fixed:
+
+    * An issue with the driver and devicetree symbol for the external flash memory where the driver was sometimes NULL, even if the DT node was chosen.
+
+MCUboot
+=======
+
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``cfec947e0f8be686d02c73104a3b1ad0b5dcf1e6``, with some |NCS| specific additions.
+
+The code for integrating MCUboot into |NCS| is located in the :file:`ncs/nrf/modules/mcuboot` folder.
+
+The following list summarizes both the main changes inherited from upstream MCUboot and the main changes applied to the |NCS| specific additions:
+
+* Added:
+
+  * :kconfig:option:`CONFIG_USE_NRF53_MULTI_IMAGE_WITHOUT_UPGRADE_ONLY` Kconfig option to specify that you want to use nRF53 multi-image upgrade without the upgrade only setting in MCUboot.
+    Enabling this option has drawbacks.
+    See the :ref:`ug_nrf5340` user guide for details.
+  * :kconfig:option:`CONFIG_PM_EXTERNAL_FLASH_HAS_DRIVER` Kconfig option.
+    When set, it makes MCUboot fail to boot when the external flash memory is used for non-primary application images with the driver for the external flash memory not enabled.
+    See :ref:`ug_bootloader_external_flash` and :ref:`pm_external_flash` for details.
+
+Zephyr
+======
+
+.. NOTE TO MAINTAINERS: All the Zephyr commits in the below git commands must be handled specially after each upmerge and each NCS release.
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``cd16a8388f71a6cce0cea871f75f6d4ac8f56da9``, with some |NCS| specific additions.
+
+For the list of upstream Zephyr commits (not including cherry-picked commits) incorporated into nRF Connect SDK since the most recent release, run the following command from the :file:`ncs/zephyr` repository (after running ``west update``):
+
+.. code-block:: none
+
+   git log --oneline cd16a8388f ^45ef0d2
+
+For the list of |NCS| specific commits, including commits cherry-picked from upstream, run:
+
+.. code-block:: none
+
+   git log --oneline manifest-rev ^cd16a8388f
+
+The current |NCS| main branch is based on revision ``cd16a8388f`` of Zephyr.
+
+Additions specific to |NCS|
+---------------------------
+
+* |no_changes_yet_note|
+
+zcbor
+=====
+
+* |no_changes_yet_note|
+
+Trusted Firmware-M
+==================
+
+* Added:
+
+  * Reading, updating, and attesting the lifecycle state stored in nRF OTP.
```

Comment:
We can add something like: 
Supporting provisioning as described  in the Platform Security Architecture (PSA) security model.

Reply:
Added